<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screen Translator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="shadowFrame">
      <property name="styleSheet">
       <string notr="true">#shadowFrame {
	background-color: rgb(46,47,48);
	border: none;
}

QTextEdit {
    background-color: rgb(64,65,66);
    color: white;
    border: none;
    border-radius: 5px;
    padding: 8px; 
}

QComboBox {
    border: 1px solid rgba(255, 255, 255, 40); /* легкий контур */
    background: transparent;
    color: white;
    padding-left: 6px;     /* ⬅️ отступ перед текстом */
    padding-right: 24px;   /* ➡️ запас для стрелки и отступа после неё */
    border-radius: 4px;
}

QComboBox:hover {
    background-color: rgba(255, 255, 255, 20); /* фон при наведении */
}

QComboBox::drop-down {
    border: none;
    background: transparent;
    width: 18px; /* шире, чтобы была зона отступа справа от стрелки */
}

QComboBox::down-arrow {
    image: url(:/icons/resources/icons/comboboxArrow.png);
    width: 12px;
    height: 12px;
}


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="3">
        <widget class="DraggableFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>38</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame
{
	border-bottom: 5px solid rgb(99,99,99);
	border: none;
	background-color: rgb(35,35,35);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="settingsButton">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	border:none;
	image: url(:/icons/resources/icons/settings.png);
}
QPushButton:hover
{
	border-radius: 3px;
	background-color: rgb(60,60,60);
	border:none;
	image: url(:/icons/resources/icons/settings_hover.png);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>268</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>30</width>
                <height>26</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../res.qrc">:/icons/resources/icons/logo.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>SF Pro Display</family>
                <pointsize>17</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(206,206,206);</string>
              </property>
              <property name="text">
               <string>Screen Translator</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>268</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="minimumSize">
               <size>
                <width>38</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>38</width>
                <height>38</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">
QPushButton
{
	image: url(:/icons/resources/icons/collapse.png);
	border: none;
}
QPushButton:hover
{
	image: url(:/icons/resources/icons/collapse.png);
	background-color: rgb(232, 17, 25);
	border: none;
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="closeButton">
              <property name="minimumSize">
               <size>
                <width>38</width>
                <height>38</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>38</width>
                <height>38</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">
QPushButton
{
	image: url(:/icons/resources/icons/close.png);
	border: none;
}
QPushButton:hover
{
	image: url(:/icons/resources/icons/close.png);
	background-color: rgb(232, 17, 25);
	border: none;
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QFrame" name="frame_4">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item row="0" column="3" rowspan="2">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>26</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="copy_1Button">
               <property name="minimumSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
	border:none;
	image: url(:/icons/resources/icons/copy.png);
}
QPushButton:hover
{
	border-radius: 3px;
	background-color: rgb(60,60,60);
	border:none;
	image: url(:/icons/resources/icons/copy.png);
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="translateButton">
               <property name="minimumSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
	border:none;
	image: url(:/icons/resources/icons/translate.png);
}
QPushButton:hover
{
	border-radius: 3px;
	background-color: rgb(60,60,60);
	border:none;
	image: url(:/icons/resources/icons/translate.png);
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="3">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>132</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>656</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" rowspan="2" colspan="3">
           <widget class="QTextEdit" name="textEdit">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QComboBox" name="translateFrom">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="clearButton">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
		color: rgb(255, 255, 255);
		
		
	
		
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 108, 107, 255), stop:1 rgba(234, 100, 100, 255));

		border-radius: 2px;
        border: 1px solid rgb(234,45,45);
        padding: 4px;
}

QPushButton:hover
{
		color: rgb(255, 255, 255);
		
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 76, 75, 255), stop:1 rgba(234, 56, 56, 255));
		border-radius: 2px;
        border: 1px solid rgb(234,45,45);
        padding: 4px;
}</string>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>304</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="switchButton">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>24</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border:none;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>304</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QFrame" name="frame_3">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QComboBox" name="translateTo">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>657</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>26</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="copy_2Button">
               <property name="minimumSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
	border:none;
	image: url(:/icons/resources/icons/copy.png);
}
QPushButton:hover
{
	border-radius: 3px;
	background-color: rgb(60,60,60);
	border:none;
	image: url(:/icons/resources/icons/copy.png);
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" rowspan="2" colspan="2">
           <widget class="QTextEdit" name="textEdit_2">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>167</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DraggableFrame</class>
   <extends>QFrame</extends>
   <header>CustomWidgets/draggableframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
